src/openrct2-ui/windows/RideList.cpp

```diff
@@ -235,7 +228,14 @@ namespace OpenRCT2::Ui::Windows
                 case WIDX_CLOSE:
                     Close();
                     break;
-                case WIDX_SORT:
+                case WIDX_HEADER_NAME:
+                    // TODO: click twice for descending sort
```

Comment:
In this PR or not?

Reply:
No, that'll be a fair bit more work, unfortunately.